### PR TITLE
Service/NWM: add nwm services

### PR DESCRIPTION
**Pull request by mailwl**

### CODE REVIEW

File: src/core/hle/service/nwm/nwm.h [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#pragma once
+
+namespace Service {
+namespace NWM {
+
+/// Initialize the NWM service
```
Comment (lioncash) [inline src/core/hle/service/nwm/nwm.h]: `Initializes all NWM services`

File: src/core/hle/service/nwm/nwm_cec.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "core/hle/service/service.h"
+
+namespace Service {
+namespace NWM {
+
+class NWM_CEC : public Interface {
```
Comment (lioncash) [inline src/core/hle/service/nwm/nwm_cec.h]: This can be a final class.

File: src/core/hle/service/nwm/nwm_ext.h [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+#include "core/hle/service/service.h"
+
+namespace Service {
+namespace NWM {
+
+class NWM_EXT final : public Interface {
+public:
+    NWM_EXT();
+    ~NWM_EXT() override;
```
Comment (lioncash) [inline src/core/hle/service/nwm/nwm_ext.h]: This destructor doesn't need to be present.

**Comment by lioncash** [comment]
Also is it intended that the commit author is  Владимир Русских (Vladimir Russian), rather than 'mailwl'?

**Comment by mailwl** [comment]
fixed. rebased to change commit author to github one

File: src/core/CMakeLists.txt [hunk 4]
```diff
@@ -124,7 +124,14 @@ set(SRCS
             hle/service/nim/nim_s.cpp
             hle/service/nim/nim_u.cpp
             hle/service/ns_s.cpp
-            hle/service/nwm_uds.cpp
+            hle/service//nwm/nwm.cpp
```
Comment (jroweboy) [inline src/core/CMakeLists.txt]: how did these all end up with two / ? seems out of place

Comment (mailwl) [inline src/core/CMakeLists.txt]: fixed

**Pull request merged**